apps/hash-graph/bin/cli/src/main.rs

```diff
@@ -5,27 +5,33 @@ mod args;
 mod error;
 mod subcommand;
 
+use std::sync::Arc;
+
 use error_stack::Result;
 use graph::load_env;
 
-use self::{args::Args, error::GraphError, subcommand::Subcommand};
+use self::{args::Args, error::GraphError};
 
-#[tokio::main]
-async fn main() -> Result<(), GraphError> {
+fn main() -> Result<(), GraphError> {
     load_env(None);
 
     let args = Args::parse_args();
 
-    match args.subcommand {
-        Subcommand::Server(args) => subcommand::server(args).await,
-        Subcommand::Migrate(args) => subcommand::migrate(args).await,
-        Subcommand::TypeFetcher(args) => subcommand::type_fetcher(args).await,
-        Subcommand::Completions(ref args) => {
-            subcommand::completions(args);
-            Ok(())
-        }
-        Subcommand::Snapshot(args) => subcommand::snapshot(args).await,
-        #[cfg(all(hash_graph_test_environment, feature = "test-server"))]
-        Subcommand::TestServer(args) => subcommand::test_server(args).await,
-    }
+    let _sentry = sentry::init(sentry::ClientOptions {
```

Comment:
For what the `_sentry` variable is used? If this is a guard we should name it like this and/or add a comment.

Reply:
Added documentation in [8e62433](https://github.com/hashintel/hash/pull/3091/commits/8e624331872d7bb6331f7e7bfd70a877aa182e66)